galloverse_ics3u_b.py

```diff
@@ -679,12 +691,64 @@ def linear_interpolation(x, x0, x1, y0, y1):
     # Rather than screen.fill, draw a rectangle
     screen.blit(bg_gallo, (x, y))
 
+    screen.blit(welcome_text_gallo, (
+        x + width // 2 - welcome_text_gallo.get_width() // 2, y + height // 3 - welcome_text_gallo.get_height() // 2))
+    scaled_text = pygame.transform.scale(text_gallo, (
+        text_gallo.get_width() * text_scale_gallo, text_gallo.get_height() * text_scale_gallo))
+    screen.blit(scaled_text,
+                (x + width // 2 - scaled_text.get_width() // 2, y + height // 2 - scaled_text.get_height() // 2))
+# ----------------------------------------------------------------------------------------
+
+    lilia_i += 15
+    if lilia_i > 300:
+        lilia_i = 0
     screen.blit(welcome_text_gallo, (x + width//2 - welcome_text_gallo.get_width()//2, y + height//3 - welcome_text_gallo.get_height()//2))
     scaled_text = pygame.transform.scale(text_gallo, (text_gallo.get_width() * text_scale_gallo, text_gallo.get_height() * text_scale_gallo))
     screen.blit(scaled_text, (x + width//2 - scaled_text.get_width()//2, y + height//2 - scaled_text.get_height()//2))
     
 
 
+    # Update the row of stars
+    star_i += 15
+    if star_i > WIDTH:
+        star_i = 0
+    # Rather than screen.fill, draw a rectangle
+    screen.fill((0, 0, 0))
```

Comment:
Can't use screen.fill, need to draw a rectangle like in the example I gave. screen.fill will cover everyone's drawings.